active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,591 @@
+
+A teaching tip can be used without a pointer when the information that it is presenting is not specific to a UI element. 
+
+<a href="url"><img src="TeachingTipSampleApp.jpg" height="600"></a>
+
+A teaching tip can require manual acknowledgement before dismissing or a teaching tip may also be light-dismiss enabled so that it will dismiss when a user scrolls or interacts with other elements of the application.
+
+<a href="url"><img src="TeachingTipLightDismissSample.jpg" height="600"></a>
+
+
+## Create a teaching tip
+
+<!-- Use the Password property to get or set the contents of the PasswordBox. You can do this in the handler for the PasswordChanged event to perform validation while the user enters the password. Or, you can use another event, like a button Click, to perform validation after the user completes the text entry. -->
+
+Here's the XAML for a teaching tip control that demonstrates the default look of the TeachingTip with a title and subtitle. 
+
+When the user clicks the button, a teaching tip will appear and display a message to the user. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to.">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   AutoSaveTip.IsOpen = true;
+}
+```
+
+Here's the result when this code runs and the clicks the button.
+
+<a href="url"><img src="TeachingTipSampleApp.jpg" height="600"></a>
+
+### Add a margin 
+
+A teaching tip can have a margin added by setting the Margin property. When a teaching tip is attached to a UI element, a margin will move it away from the UI element it is pointing to. When a teaching tip is not attached to anything, a margin will move it away from the side of the app window it is positioned towards. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="80">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   AutoSaveTip.IsOpen = true;
+}
+```
+
+<a href="url"><img src="TeachingTipMarginSample.jpg" height="600"></a>
+
+### Point to a UI element
+
+You can change the teaching tip to point to a specific UI element by setting the TeachingTipPlacementMode property. For a targeted teaching tip, this placement will be relative to the UI element target. For a non-targeted teaching tip, this placement will relative to app window.  
+
+XAML
```

Comment:
This code example doesn't actually use the TeachingTipPlacementMode property. And based on the name, I would expect this property to do something other than the section title (Point to a UI element). Flyout's placement mode is the positioning (above, below, etc) in XY space relative to a target, but it's not really related to pointing (e.g. with a beak).

Reply:
You're right, this example and others should include examples of setting the SetAttach property - though it looks like we are also questioning whether this is the best approach for attaching tips.

Reply:
Have you considered a Target property, similar to FlyoutBase?

Reply:
The Spec has now been updated to use the Target property. Thanks again, @YuliKl!